JavaScript/Topics/7.Execution_Context/README.md

```diff
@@ -1,6 +1,61 @@
 # Execution Context in JavaScript
 
-When running code, **Execution Context** gets created by our JS engine, which does all the magic in the background. Here's how!
+Before diving deep into what the execution context is we have to first understand what the javascript engine is because it plays a crucial role and performs a lot of tasks before the role of execution context came.
+
+# What is Javascript Engine?
+Javascript Engine is nothing but a program that executes our javascript code. Every browser has its own Javascript engine but probably the most well-known engine is Google's V8.
```

Comment:
Use " - " (hyphen) before the sentence to make it a bullet point.
Eg:
- JavaScript
Hope you get it:)